wp_api/src/lib.rs

```diff
@@ -212,6 +215,32 @@ pub enum IntegerOrString {
     String(String),
 }
 
+#[uniffi::export(with_foreign)]
+#[async_trait::async_trait]
+pub trait WpAppNotifier: Send + Sync + std::fmt::Debug {
+    async fn unauthorized_request(&self);
```

Comment:
Should this be more specific, like `authentication_becomes_invalid`?

Reply:
`becomes_invalid` may be a little misleading because we currently don't check if the authentication is valid when we first build the client. Maybe we should go with `requested_with_invalid_authentication`?

Reply:
Sounds good to me!

Reply:
Addressed in a93523cd4064b3adfa52ea0ef3d61120152be51d.